rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
Seems like a good way to do it. You'll need to update the api-service.md doc to match the new option sematics, as I commented inline there. After that this LGTM and I think is ready to merge unless jonboulle objects.